services/proctoring-service/.env

```
@@ -1,22 +1,9 @@
-# URL của Ganache service (trong Docker hoặc chạy local)
+PORT=8082
 WEB3_PROVIDER_URL=http://127.0.0.1:7545
-
-# Private Key của tài khoản Owner (lấy từ Ganache)
-ACCOUNT_PRIVATE_KEY=0x7f948ca9326dda3aa3c7dae71d8150af2e1f7753cc8d70c89c30ee85f9aeec2b
-
-# Địa chỉ của Smart Contract sau khi deploy (sẽ điền sau)
 CONTRACT_ADDRESS=0x623f718db453a5977F789aEb1af97042510801E7
-JWT_SECRET=mySecretKey12345678901234567890123456789012345678901234567890
\ No newline at end of file
+ACCOUNT_PRIVATE_KEY=0xed09b80beb745076e9134be4f4015447c7272f3d73a7712d707eb257c35a0099
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ACCOUNT_PRIVATE_KEY key should go before the CONTRACT_ADDRESS key

(UnorderedKey)